### ISSUE TITLE
prev_next partial broken for hugo > 0.55.x

### ISSUE THREAD

**Issue by dudash** [RedHatGov/redhatgov.github.io#241]
We are currently using a pretty old version of hugo so this doesn't affect the website. However, we will probably want to upgrade at some point past 0.55.x so logging this as a bug

See error details below:
```
MacBook-Pro:redhatgov dudash$ ./bin/hugo-0.65.1.mac server
Building sites … ERROR 2020/02/20 13:11:50 render of "page" failed: execute of template failed: template: _default/lab.html:14:7: executing "_default/lab.html" at <partial "prev_next_links" .Page>: error calling partial: "/Users/dudash/Documents/SourceCode/redhatgov/layouts/partials/prev_next_links.html:13:28": execute of template failed: template: partials/prev_next_links.html:13:28: executing "partials/prev_next_links.html" at <$prevLab.Permalink>: nil pointer evaluating page.Page.Permalink
ERROR 2020/02/20 13:11:50 render of "page" failed: execute of template failed: template: _default/lab.html:14:7: executing "_default/lab.html" at <partial "prev_next_links" .Page>: error calling partial: "/Users/dudash/Documents/SourceCode/redhatgov/layouts/partials/prev_next_links.html:13:28": execute of template failed: template: partials/prev_next_links.html:13:28: executing "partials/prev_next_links.html" at <$prevLab.Permalink>: nil pointer evaluating page.Page.Permalink
ERROR 2020/02/20 13:11:50 render of "page" failed: "/Users/dudash/Documents/SourceCode/redhatgov/layouts/_default/lab.html:18:28": execute of template failed: template: _default/lab.html:14:7: executing "_default/lab.html" at <partial "prev_next_links" .Page>: error calling partial: "/Users/dudash/Documents/SourceCode/redhatgov/layouts/partials/prev_next_links.html:18:28": execute of template failed: template: partials/prev_next_links.html:18:28: executing "partials/prev_next_links.html" at <$nextLab.Permalink>: nil pointer evaluating page.Page.Permalink
ERROR 2020/02/20 13:11:50 render of "page" failed: execute of template failed: template: _default/lab.html:14:7: executing "_default/lab.html" at <partial "prev_next_links" .Page>: error calling partial: "/Users/dudash/Documents/SourceCode/redhatgov/layouts/partials/prev_next_links.html:13:28": execute of template failed: template: partials/prev_next_links.html:13:28: executing "partials/prev_next_links.html" at <$prevLab.Permalink>: nil pointer evaluating page.Page.Permalink
Built in 4269 ms
Error: Error building site: failed to render pages: render of "page" failed: "/Users/dudash/Documents/SourceCode/redhatgov/layouts/_default/lab.html:18:28": execute of template failed: template: _default/lab.html:14:7: executing "_default/lab.html" at <partial "prev_next_links" .Page>: error calling partial: "/Users/dudash/Documents/SourceCode/redhatgov/layouts/partials/prev_next_links.html:18:28": execute of template failed: template: partials/prev_next_links.html:18:28: executing "partials/prev_next_links.html" at <$nextLab.Permalink>: nil pointer evaluating page.Page.Permalink
```

**Comment by ajacocks** [comment]
Thanks, Jason. We've been needing to do this, for a while now.

- Alex

**Comment by kenmoini** [comment]
Can close as fix'd.

**Comment by dudash** [comment]
fixed by PR #281
